cogs/guess.py

```diff
@@ -0,0 +1,97 @@
+    # cogs/guess.py:8:0: R0903: Too few public methods (1/2) (too-few-public-methods)
+    '''this is guessing game'''
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="guess", pass_context=True)
+    async def mycom(self, context):
+        '''pylint told me to do this, this will start a guessing game :aquaThumbsUp:'''
+        # welcoming the user
+        await self.bot.reply("guess what word I'm thinking of!")
+
+        # thinking
+        await asyncio.sleep(3)
+
+        # start guess
+        await self.bot.say("Start guessing...(hint: no caps)")
+        await self.bot.say("you have 30 seconds to make a guess")
+        await asyncio.sleep(0.5)
+
+        # make secret
+        secret = "its gonna change"
+
+        # Probably need to create a word list and grab words from there
+        items = ["here", "are", "some", "strings", "of", "which", "we", "will", "select", "one",
+                 "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", "python",
+                 "ruby", "java", "javascript", "traceback", "print", "random", "input", "raw",
+                 "print", "sleep", "time", "secret", "while", "then", "if", "loop", "in", "else",
+                 "break", "empty", "mocha", "matcha", "money", "spam", "jazz", "jam", "spaz",
+                 "binder", "jokes", "chemistry", "person", "matcha", "night", "everyone", "pasta",
+                 "craving", "jolly", "ranch", "lit", "still", "first", "second", "third", "skull",
+                 "ketchup", "ice", "red", "life", "death", "magic", "comments", "class", "secret",
+                 "value", "variable", "guess", "incorrect", "russian", "roulette", "bot", "turn",
+                 "failed", "wood", "stone", "cobblestone", "blanket", "ore", "chest", "poster",
+                 "hash", "secure", "magical", "never", "public", "private", "key", "encryption",
+                 "cypher", "cryptogram", "theory", "open", "thread", "ripper", "core", "graphical",
+                 "graphics", "asymmetrical", "symmetrical"]
+        # picks one word from list
+        secret = random.choice(items)
+
+        # creates a variable with an empty value
+        guesses = ''
+        turns = 5
+        while turns > 0:
+            failed = 0
+            for char in secret:
+                if char in guesses:
+                    await self.bot.say(char)
+                else:
+                    await self.bot.say("_")
+                    failed += 1
+                #if failed is equal to zero
+                # Win text
+            if failed == 0:
+                await self.bot.say("You won! Good job! :aquaThumbsUp:")
+                # exit the script"""
+                break
+            # ask the user go guess a character"""
+            await self.bot.say("lower case letters please~~!")
+            guess = self.bot.wait_for_message(channel=context.message.channel)
```

Comment:
You'll want an extra parameter here: `timeout=30`.  After 30 seconds, if we don't get a response,  `wait_for_message` will return None, and it can be used with the check I mentioned.